gradle.properties

```diff
@@ -1,2 +1,2 @@
 group=dev.sigstore
-version=0.1-SNAPSHOT
+version=dev-SNAPSHOT
```

Comment:
dev?

Reply:
yeah I dunno, I was thinking we avoid the whole dance of constantly updating this version. We already have commits anyway, this number is much less relevant now.